TASK/AddPatrolRouteLink.md

```diff
@@ -5,11 +5,26 @@ ns: TASK
 
 ```c
 // 0x23083260DEC3A551 0xD8761BB3
-void ADD_PATROL_ROUTE_LINK(Any p0, Any p1);
+void ADD_PATROL_ROUTE_LINK(int id1, int id2);
 ```
 
+connects 2 route nodes to make it clear how the ped should move
 
 ## Parameters
-* **p0**: 
-* **p1**: 
+* **id1**: 
+* **id2**: 
 
+## Examples
+![image](https://user-images.githubusercontent.com/55803068/188470866-c32c6a9f-a25d-4772-9b18-5be46e2c14a1.png)
+
+``` lua
```

Comment:
Maybe I'm just nitpicking here, but could you please remove the space between the backticks and lua?

Replace with:
```
```lua
```

Thanks! 🙂